WordPressAuthenticator/GoogleSignIn/URL+GoogleSignIn.swift

```diff
@@ -19,17 +19,8 @@ extension URL {
             ("redirect_uri", clientId.defaultRedirectURI),
             ("response_type", "code"),
             // TODO: We might want to add some of these or them configurable
-            //
-            // The request we make with the SDK asks for:
-            //
-            // - email
-            // - profile
-            // - https://www.googleapis.com/auth/userinfo.email
-            // - https://www.googleapis.com/auth/userinfo.profile
-            // - openid
-            //
             // See https://developers.google.com/identity/protocols/oauth2/scopes
-            ("scope", "https://www.googleapis.com/auth/userinfo.email")
+            ("scope", "openid https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/userinfo.profile")
```

Comment:
I haven't tried it yet, but I think we could remove `https://www.googleapis.com/auth/userinfo.profile`. The only information we need other than the id token from Google is the email adderss.

Reply:
It appears [GoogleSignIn SDK uses "email profile" as the scopes](https://github.com/google/GoogleSignIn-iOS/blob/7.0.0/GoogleSignIn/Sources/GIDScopes.m#L58-L61).

Reply:
Great find, thanks!

The code also [suggests](https://github.com/google/GoogleSignIn-iOS/blob/7932d33686c1dc4d7df7a919aae47361d1cdfda4/GoogleSignIn/Sources/GIDScopes.m?rgh-link-date=2023-02-07T22%3A08%3A56Z#L19-L22) that the "https://..." scopes are no longer needed as "old".

I'll try this in a PR later on.